docs/manual/background.md

```diff
@@ -5,24 +5,34 @@ to fall cleanly into one of two camps: package-based or image-based.
 
 ### Package system benefits and drawbacks
 
- * + Highly dynamic, fast access to wide array of software
- * + State management in `/etc` and `/var` is well understood 
- * + Can swap between major/minor system states (`apt-get upgrade` is similar to `apt-get dist-upgrade`)
- * + Generally supports any filesystem or partition layout
+Benefits
+
+ * Highly dynamic, fast access to wide array of software
+ * State management in `/etc` and `/var` is well understood
+ * Can swap between major/minor system states (`apt-get upgrade` is similar to `apt-get dist-upgrade`)
+ * Generally supports any filesystem or partition layout
+
+Drawbacks
+
  * - As package set grows, testing becomes combinatorially more expensive
  * - Live system mutation, no rollbacks
```

Comment:
Looks like this needs the same treatment, right? I.e.

```suggestion
 * As package set grows, testing becomes combinatorially more expensive
 * Live system mutation, no rollbacks
```

?

Reply:
oops, yes!